draft-regext-rdap-extensions.md

```diff
@@ -522,52 +682,41 @@ following:
-of the content within the JSON, and all extensions identifiers MUST be free from conflict in
-both their syntactic and semantic meaning.
-
-Note that this document does not update the guidance from [@!RFC9083, Section 4.1] regarding
-"help" responses and the `rdapConformance` array.
+The primary purpose of an RDAP extension specification is to aid in
+the implementation of RDAP clients. Extension authors should consider
+the following content guidelines:
 
-When a server implementation supports multiple extensions, it is RECOMMENDED that the server
-also support and return versioning information such as that defined by [@?I-D.gould-regext-rdap-versioning].
+1. Examples of RDAP JSON should be generously given, especially in
+areas of the specification which may be complex or difficult to describe
+with prose.
+2. Normative references, i.e. references to materials that are
+required for the interoperability of the extension, should be stable
+and non-changing.
+3. Extension specifications should strongly consider making the use
+of HTTPS with RDAP mandatory if appropriate.
 
-# Extension Definitions
+## Extension Definitions
 
-Extensions must be documented in an RFC or in some other permanent and readily
-available reference, in sufficient detail that interoperability between independent 
-implementations is possible.
+Extensions must be documented in an RFC or in some other permanent and
+readily available reference, in sufficient detail that
+interoperability between independent implementations is possible.
 
-Though RDAP gives each extension its own namespace, the definition of an
-extension may re-use definitions found in the base RDAP specification or in any
-other properly registered extension.
+Though RDAP gives each extension its own namespace, the definition of
+an extension may reuse definitions found in the base RDAP
+specification or in any other registered extension.
 
-[@!RFC9083] notes that the extension identifiers provide a "hint" to the client
-as to how to interpret the response. This wording does not intentionally restrict
-the extension to defining only JSON values within the extension's namespace.
-Therefore, an extension may define the use of its own JSON values together
-with the use of JSON values from other extensions or RDAP specifications. As with
-the ICANN profile or NRO profile extensions, the extension may simply signal 
-policy applied to already defined RDAP structures.
+[@!RFC9083] notes that the extension identifiers provide a "hint" to
+the client as to how to interpret the response. This wording does not
+intentionally restrict the extension to defining only JSON values
+within the extension's namespace.  Therefore, an extension may define
+the use of its own JSON values together with the use of JSON values
+from other extensions or RDAP specifications. As with the ICANN
+profile or NRO profile extensions, the extension may simply signal
```

Comment:
Since not everyone would know about them, should we include informative references to the ICANN and NRO profile extensions?

Reply:
Yep, makes sense, updated.